examples/with-graphql-amplication/styles/footer.js

```diff
@@ -0,0 +1,60 @@
+import css from 'styled-jsx/css'
+
+export const footer = css.resolve`
```

Comment:
This might as well just use CSS modules, yeah?

Reply:
Done!